index.html

```diff
@@ -1741,7 +1975,6 @@ <h4 class="modal-title">Help</h4>
                         return;
                     }
                     var url = window.location.host;
-                    if (url.indexOf("playtak") > -1)
```

Comment:
These Debugging clamps prevented the offline-client from reaching the server. I removed them in order to connect. This may or may not be useful in the merge.